spog/api/src/lib.rs

```diff
@@ -40,6 +42,26 @@ pub struct Run {
     pub(crate) infra: InfrastructureConfig,
 }
 
+impl Default for Run {
+    fn default() -> Self {
+        Self {
+            snyk: Snyk { org: None, token: None },
+            bind: "127.0.0.1".to_string(),
+            port: 8084,
+            guac_url: "http://localhost:8080/query".to_string(),
+            sync_interval_seconds: 10,
+            bombastic_url: Url::from_str("http://localhost:8082").unwrap(),
+            vexination_url: Url::from_str("http://localhost:8081").unwrap(),
```

Comment:
If we're gonna hardcode the ports here, then I'd like to remove the ability to override the ports with options.